docs/src/pages/Homepage.vue

```diff
@@ -23,28 +23,48 @@
       <br v-if="$q.screen.gt.sm" />to navigate through our most important
       technical resources.
     </p>
-    <h2
-      class="landing-heading landing-heading--medium q-mb-lg normal-line-height"
-    >Have a look at Our most used pages</h2>
-    <div class="cards-grid landing-mb--large">
-      <q-card
-        v-for="({path, label, icon}, pageIndex) in mostUsedPages"
-        :key="pageIndex"
-        class="raise-on-hover card column justify-end items-center cursor-pointer"
-        flat
-        background="grey-10"
-        @click="$router.push(`/${path}`)"
-      >
-        <q-icon
-          :name="icon"
-          class="card__icon q-mb-lg"
-          color="brand-primary"
-        />
-
-        <div class="q-pa-sm q-mb-md text-center text-brand-secondary text-size-16 text-weight-bold letter-spacing-300">
-          {{ label }}
+    <div :class="{'cards-grid': $q.screen.gt.sm}">
+      <div class="row cards-container-width justify-center q-gutter-sm q-py-lg justify-self-end">
+        <div class="text-center full-width text-size-24 text-weight-bolder letter-spacing-450 text-brand-primary text-uppercase">
+          Most Used
+        </div>
+        <q-card
```

Comment:
You removed the `@click`, so these cards aren't working as link as they should (do it using router-link)